README.md

```diff
@@ -171,3 +193,26 @@ Example output (timing scale: sec.):
     std::sort        0.402458
     std::stable_sort 2.436326
     timsort          0.298639
+
+Example bench_merge output (timing scale: milliseconds; omitted detailed results for different
+middle iterator positions, reformatted to improve readability):
+
+    c++ -v
+    Using built-in specs.
+    ...
+    Target: x86_64-pc-linux-gnu
+    ...
+    gcc version 10.2.0 (GCC)
+    c++ -I ../include -Wall -Wextra -g -DNDEBUG -O2 -std=c++11 bench_merge.cpp -o bench_merge
+    ./bench_merge
+    middle\algorithm:    	std::inplace_merge	timmerge
+    size	100000
+    RANDOMIZED SEQUENCE
+    [int] AVERAGE        	 17.163368        	 16.056263
```

Comment:
Doesn't this go against the claim that timmerge is slower for small simple types?

Reply:
It kind of does. But when you run the benchmark and look at the detailed *RANDOMIZED int* table, you'll notice that `timmerge` wins only when the `middle` is very close to the beginning or the end. Unfortunately, the `middle` points are not uniformly distributed across the entire range of elements. And I think this distribution should not be uniform despite the misleading average result, because the numbers vary most at the range edges. I welcome suggestions how to make the average numbers less misleading. Give them weights? Or list the `middle == size/2` numbers in addition to the average numbers?

```
size	100000
RANDOMIZED SEQUENCE
[int]
middle\algorithm:	std::inplace_merge	timmerge
         0       	  0.098000        	  0.097000
         1       	  6.451000        	  1.116000
         2       	  6.410000        	  1.058000
         5       	 14.302000        	  2.326000
       100       	 14.370000        	  3.505000
      1000       	 15.291000        	 10.783000
      5000       	 19.183000        	 28.570000
     20000       	 33.082000        	 36.837000
     33333       	 43.424000        	 47.126000
     50000       	 48.645000        	 50.975000
     75000       	 33.969000        	 40.889000
     85714       	 24.553000        	 30.223000
     96000       	 14.904000        	 29.419000
     98901       	 12.195000        	 12.690000
     99915       	 10.924000        	  3.346000
     99992       	 10.629000        	  2.233000
     99998       	  8.777000        	  1.871000
     99999       	  8.799000        	  1.900000
    100000       	  0.098000        	  0.105000
   AVERAGE       	 17.163368        	 16.056263
```

Reply:
Alternatively we could have minimal benchmarks results in the README, and direct readers to a wiki page with more complete benchmarks if they want the whole story.

Reply:
* Replaced AVERAGE with approx. average based on [Trapezoidal rule for approximating the definite integral](https://en.wikipedia.org/wiki/Trapezoidal_rule).
* Added a Wiki page with detailed results and linked to it in the README.